draft-barnes-mimi-arch.md

```diff
@@ -320,27 +355,26 @@ The overall MIMI protocol defines this event framework, including its
 authentication scheme, as well as the mechanics of how events are delivered from
 one server to another.
 
-## Control Protocols
+## Room State Synchronization
 
-The servers involved in a room use control protocols to perform actions related
-to different types of information that comprise a room's state, particularly
-those listed in {{room-state}}.  Because these types of information and the
-operations they require are largely orthogonal, it makes sense to have a
-separate control protocol for each type of information.
+The servers involved in a room use an application state synchronization protocol
+to coordinate changes to a room's state, particularly those listed in
+{{room-state}}.  A few types of room state are synchronized, in what can be
+viewed as independent control sub-protocols:
 
-The **policy control protocol** distributes information about the policy
+A **policy control protocol** distributes information about the policy
 envelope of a room, and allows participants in a room to propose changes to the
 policy within that envelope.
 
-The **participation control protocol** manages the user-level membership of the
+A **participation control protocol** manages the user-level membership of the
 room, including the various ways that members might join or leave a room (or be
 added/removed by other users).
 
-The **end-to-end security control protocol** manages the end-to-end security
-state of the room.  In addition to distributing messages that add or remove
-clients from the end-to-end security state, this protocol also allows servers
-to distribute cryptographic information that clients have pre-registered, which
-allows clients to be asynchronously added to rooms.
+As discussed above, the **end-to-end security control protocol** manages the
+end-to-end security state of the room.  In addition to distributing messages
+that add or remove clients from the end-to-end security state, this protocol
+also allows servers to distribute cryptographic information that clients have
+pre-registered, which allows clients to be asynchronously added to rooms.
```

Comment:
That last sentence is a bit long. Maybe split off the last bit? Something like "...that clients have pre-registered. That information in turn allows the asynchronous addition of clients to rooms.